src/spell.c

```diff
@@ -1902,6 +1975,80 @@ spell_load_lang(char_u *lang)
 #endif
     }
 
+    if (r == FAIL)
+    {
+#	define HUNSPELLDICT "/usr/share/myspell/"
```

Comment:
A hard-coded path to the dictionaries is not acceptable.

Reply:
Right, any idea how to discover this location automagically (I wouldn't like to add one more option to ``configure``)? Do we need to parse output of ``echo|hunspell -D``?

Reply:
Not sure if I get things right, but could this be made configurable at run-time? And if not configured at run-time, then use the default from hunspell (which maybe could mean `echo|hunspell -D` parsing at build-time, if there is no other/better way).